tools/android/json_worker_wrapper.py

```diff
@@ -0,0 +1,61 @@
+# pylint: disable=g-direct-third-party-import
+# Copyright 2023 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A wrapper for AAR extractors to support persistent worker."""
+
+import json
+import sys
+import traceback
+
+
+def wrap_worker(flags, main, app_run):
+  """
+
+  Args:
+    flags: The FlagValues instance to parse command line arguments.
+    main: The main function to execute.
+    app_run: app.run() function in Abseil.
+  """
+  if sys.argv[1] == "--persistent_worker":
```

Comment:
Continuing from here: https://github.com/bazelbuild/bazel/pull/18496#discussion_r1213869239

I think this logic is fine, but it's unclear to me if the features we were discussing in the above linked thread (profiling, pdb) will work if you call through `wrap_result()` as opposed to `app_run()`. To safeguard against this, let's add a check inside this persisent worker condition that basically asserts, "If persistent worker mode AND one of {pdb, profiling, cprofile} are requested in FLAGS, exit with an error code".